packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -262,6 +262,14 @@ async function checkFinality (transfer) {
 // on the Near2EthClient.
 async function checkSync (transfer) {
   const web3 = new Web3(getEthProvider())
+  const ethNetwork = await web3.eth.net.getNetworkType()
+  if (ethNetwork !== process.env.ethNetworkId) {
```

Comment:
Do we have lot of dependencies on environment variables already? The defaults are information that we [want to expose](https://github.com/near/rainbow-bridge-frontend/pull/75#issuecomment-770103986), though we can allow overriding with environment variables. 

On a call last week, we discussed adding `ethNetwork` and `nearNetwork` to each transfer. I'm curious why you ultimately decided against this approach. It seems like it might simplify #6 to add it, where a transfer gets `ethNetwork` set to `await web3.eth.net.getNetworkType()` during initialization, and can then check that `web3.eth.net.getNetworkType()` hasn't changed in all the spots modified in this PR. It seems clean to me to avoid referencing environment variables where possible, though I haven't thought deeply about that yet.

Reply:
Yes, the nep141-erc20 library uses `process.env` for accessing the bridge context in many places: in unlock() with `process.env.ethLockerAddress`, in checkSync() with `process.env.ethClientAddress`, in lastBlockNumber(), in Near provider initialization, in findProof(), in lock() and in deployBridgeToken().

So right now it makes more sense to me to keep this pattern when comparing the networkId. If we do decide to add bridge context in each localStorage transfer, then I believe it requires more thinking in the context of multiple bridge support (#6) and modifying the localStorage transfer object information would be better addresses at that time.